src/titiler/xarray/tests/fixtures/pyramid.zarr/.zgroup

```diff
@@ -0,0 +1,3 @@
+{
+    "zarr_format": 2
```

Comment:
would be good to do a `git rm -r` on `src/titiler/xarray/tests/fixtures/pyramid.zarr` since pyramid support has been removed

Reply:
I still test the `group` option for now